.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+  printWidth: 100,
```

Comment:
Maybe 120 or 150, that is optional

Reply:
Done.
[Resolved here](https://github.com/OldBoyHusc/react-native-multi-select/pull/2#discussion_r712649755)